apps/block_scout_web/lib/block_scout_web/schema.ex

```diff
@@ -84,6 +84,12 @@ defmodule BlockScoutWeb.Schema do
       resolve(&CeloAccount.get_by/3)
     end
 
+    @desc "Gets all the claims given a address hash."
+    field :celo_claims, list_of(:celo_claims) do
+      arg(:hash, non_null(:address_hash))
+      resolve(&CeloAccount.get_by/3)
```

Comment:
Looks like the problem is here, this call gets directed to wrong clause (line 7 at celo_account.ex).